wagtail/wagtailredirects/views.py

```diff
@@ -12,7 +12,9 @@
 from wagtail.wagtailredirects.forms import RedirectForm
 
 
-@any_permission_required('wagtailredirects.add_redirect', 'wagtailredirects.change_redirect', 'wagtailredirects.delete_redirect')
+@any_permission_required(
+    'wagtailredirects.add_redirect', 'wagtailredirects.change_redirect', 'wagtailredirects.delete_redirect'
```

Comment:
I think this would look a lot clearer if they were each on their own line (since they share a common prefix). No need to change it again though if you don't want to.